src/getters/crunchbase.py

```diff
@@ -0,0 +1,114 @@
+"""
+discovery_utils.getters.crunchbase.py
+
+Module for easy access to downloaded CB data on S3.
+
+"""
+import pandas as pd
+import boto3
+import logging
+import re
+
+
+# Define global variables
+BUCKET_NAME = "discovery-iss"
+S3_PATH = "data/crunchbase/"
+AWS_ACCESS_KEY =
+AWS_SECRET_KEY = 
+FILE_NAMES = [
+    "acquisitions.csv",
+    "category_groups.csv",
+    "checksum.csv",
+    "degrees.csv",
+    "event_appearances.csv",
+    "events.csv",
+    "funding_rounds.csv",
+    "funds.csv",
+    "investment_partners.csv",
+    "investments.csv",
+    "investors.csv",
+    "ipos.csv",
+    "jobs.csv",
+    "org_parents.csv",
+    "organization_descriptions.csv",
+    "organizations.csv",
+    "people_descriptions.csv",
+    "people.csv"
+]
+
+
+# Initialize S3 client
+S3 = boto3.client(
+    "s3", aws_access_key_id=AWS_ACCESS_KEY, aws_secret_access_key=AWS_SECRET_KEY
+    )
+
+
+# Building block functions - not to be used directly
+
+def list_s3_directories():
+    """List S3 directories under the specified prefix."""
+    # List all objects with the specified prefix
+    objects = S3.list_objects_v2(Bucket=BUCKET_NAME, Prefix=S3_PATH, Delimiter='/')
+
+    # Extract the directory names from the CommonPrefixes list
+    directories = [prefix['Prefix'] for prefix in objects.get('CommonPrefixes', [])]
+
+    return directories
+
+def extract_timestamp(directory):
```

Comment:
Could you please indicate the types of inputs and outputs? eg:
```suggestion
def extract_timestamp(directory: str) -> str:
```

Reply:
Thanks @beingkk, I've added these now - think I may have added them while you were reviewing.